packages/block-library/src/latest-posts/edit.js

```diff
@@ -168,27 +224,50 @@ class LatestPostsEdit extends Component {
 						[ `columns-${ columns }` ]: postLayout === 'grid',
 					} ) }
 				>
-					{ displayPosts.map( ( post, i ) => {
-						const titleTrimmed = post.title.rendered.trim();
-						return (
-							<li key={ i }>
-								<a href={ post.link } target="_blank" rel="noreferrer noopener">
-									{ titleTrimmed ? (
-										<RawHTML>
-											{ titleTrimmed }
-										</RawHTML>
-									) :
-										__( '(Untitled)' )
-									}
-								</a>
-								{ displayPostDate && post.date_gmt &&
-									<time dateTime={ format( 'c', post.date_gmt ) } className="wp-block-latest-posts__post-date">
-										{ dateI18n( dateFormat, post.date_gmt ) }
-									</time>
-								}
-							</li>
-						);
-					} ) }
+					{ displayPosts.map( ( post, i ) =>
+						<li key={ i }>
+							{ displayPostFeaturedImage && post._embedded[ 'wp:featuredmedia' ] &&
+								<img
+									alt={ post._embedded[ 'wp:featuredmedia' ][ 0 ].title.rendered }
+									src={ post._embedded[ 'wp:featuredmedia' ][ 0 ].source_url }
```

Comment:
@AmartyaU This can be skipped, but I've noticed that the style, in general, in rendering markup in React is to avoid such long access variables (`post._embedded[ 'wp:featuredmedia' ][ 0 ].title.rendered`)  and declare them upper in the code. It allows you to:

- use Object/Array [destructuring](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment) and have a simple short variable to use in the markup
- add a snippet of comment why on we have to use the magic zero to get the feature image

Because you are in a map() and it looks very nice maybe a helper function that handles the above could do it?

Reply:
@draganescu As mentioned in the PR description, I am only working on the 'display post content' section of the latest post block. The part you mentioned was completed by @ajitbohra in #13698.

On this note, @gziolo @getdave should I remove the parts of #13698 from my PR or submit my PR with those other parts included?

Reply:
Yes, you can remove them if they are covered in #13698.

Reply:
Yeah, I still need to revise. I was having some trouble debasing so am figuring that out